CONTRIBUTING.rst

```diff
@@ -2,226 +2,147 @@
 Contributing guidelines
 ***********************
 
-In general
-==========
-
 - `PEP 8`_, when sensible.
 - Test ruthlessly. Write docs for new features.
-- Even more important than Test-Driven Development--*Human-Driven Development*.
+- Make sure any new logic is easy for others to understand.
 - If you add an extension to setup.py, add it to supportedextensions.md.
 - Please update AUTHORS.rst when you contribute.
 
 .. _`PEP 8`: http://www.python.org/dev/peps/pep-0008/
 
-In particular
-=============
-
-
 Setting up for development
---------------------------
+==========================
 
-Clone the repo: ::
+Clone the repo:
+
+.. code-block:: bash
 
     $ git clone https://github.com/CenterForOpenScience/modular-file-renderer.git
     $ cd modular-file-renderer
 
 Configure development environment and install the development dependencies.
 
 .. note::
-
-    It is recommended that you use a `virtualenv`_ with `virtualenvwrapper`_ during development. Python 3.5 or greater, `R`_, and `pspp`_ are required.
-
-.. _virtualenv: http://www.virtualenv.org/en/latest/
-.. _virtualenvwrapper: https://pypi.python.org/pypi/virtualenvwrapper
+    Python 3.5 or greater, `R`_, and `pspp`_ are required.
+    Python 3.6 is reccomended. It's recommended that a python version manager such as pyenv is used. Additionally it is recommended that you use a virtual environment during development.
+.. _pyenv: https://github.com/pyenv/pyenv
```

Comment:
I'm more familiar with pyenv, so that's what I wrote about. If we feel we need to add two examples to really drive home the idea that managing dependency versions across projects is a best practice, I'm all for it.

Reply:
Made a different ticket for this. https://openscience.atlassian.net/browse/SVCS-840

Reply:
Sounds good